core/src/main/resources/reference.conf

```diff
@@ -430,6 +430,17 @@ cassandra-query-journal {
     # Verbose debug logging for offset updates for events by tag queries. Any logging that is per event is
     # affected by this flag. Debug logging that is per query to Cassandra is enabled if DEBUG logging is enabled
     verbose-debug-logging = false
+
+    # Set to a duration to have events by tag queries drop state about persistence ids
+    # this can be set to reduce memory usage of events by tag queries for use cases where there
+    # are many short lived persistence ids
+    # If the metadata for a persistence-id is dropped and the persistence-id is encoutered it will
+    # add a delay of the new-persistence-id-scan-timeout before delivering any new events
+    # If the query is started with NoOffset then the the stream will fail
+    # This should be set to a large value e.g. 2 x the bucket size. It can be set lower but if the metadata is dropped
+    # for a persistence id and an event is received afterward then old events in the current and previous bucket may
+    # be redelivered
+    cleanup-old-persistence-ids = "off"
```

Comment:
I wonder if we can have it enabled by default (at least in master/1.0) by having a rather long duration?

Reply:
It would be safe to set it to some multiple of the bucket size

Reply:
`2 Days` would be better than nothing.

Reply:
Set it to 2x bucket by default